.github/workflows/build.yml

```diff
@@ -69,18 +69,22 @@ jobs:
 
       # Check out current repository
       - name: Fetch Sources
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+
+      # Validate wrapper
```

Comment:
I realized it's not needed here. Let's remove it :(

Reply:
Removed